### ISSUE TITLE
Issues with accents

### ISSUE THREAD

**Issue by pauline182** [R-ArcGIS/r-bridge#23]
Hello,
I've got an issue when I'm running my R-Script on ArcMap, using R-ArcGIS bridge.
I work with a feature class stored in a geodatabase. When I run the script on R-studio, everything works perfectly such as field calculator, saving of feature class and tables in my geodatabase and some plots as images. The bridge between the two softwares works very well.
Unfortunately, when I try to import my script on ArcMap to add my process in a toolbox, it seems that ArcGIS cannot read sentences which included accents (as you can perceived by my english, I'm french and we have a lot of accentuated words). So, it's a real problem that doesn't allow me to automatise my processing on ArcGIS.
I have this issue also with specials characters such as "µm" or exponent.
Does anyone know if my issue came from ArcGIS or the bridge between R and ArcGIS?

This is my data frame on R
![image](https://user-images.githubusercontent.com/39953626/41167728-130768e8-6b44-11e8-8d88-cc2692ba6dbc.png)

And this s what ArcGIS is reading when I run my script on ArcGIS desktop
![image](https://user-images.githubusercontent.com/39953626/41167774-34836d5a-6b44-11e8-88a1-015c0dccff29.png)

It misses all my data with accents.
If someone has ever had this problem or is acquainted with a solution, it will be time saving for me,

I thank you in advance,

Best regards,

Pauline

**Comment by scw** [comment]
Hello @pauline182, sorry to hear of the issue you mentioned. Could you let us know what file formats you're writing to? Some formats (particularly shapefiles) have limitations around character encoding that have to be handled in special ways to pass strings with complex characters. Or if you could share an example dataset that exhibits the problem, that's also helpful.

Thanks,
Shaun

**Comment by pauline182** [comment]
Hello Shaun, 

My issue wasn't about writing my data. When I save my data in a feature class inside a geodatabase, I use "arc.write" function and everything work perfectly, the accents are there.
My problem was only with the library ggplot. When I exported graphics, I had a problem with accents and special characters that didn't showed up.
I thought the issue was caused by arcgisbinding and ArcGIS because when I printed my data on ArcGIS (while I was running my script on ArcGIS ) neither of the accents nor special characters appeared.
I finally was able to solve my issue only by adding two sentences on my R script to set encoding system : 
  Sys.setlocale("LC_ALL", "French")
  options(encoding = "UTF-8")
For now, my scripts using R-bridge work very well but encoding system between ArcGIS and R seems to be a little tricky sometimes.

Thanks for answering to my request,

Best regards,

Pauline
